tests/tt_eager/python_api_testing/unit_testing/misc/test_ssm_1d_sum_reduce.py

```diff
@@ -70,9 +72,12 @@ def test_ssm_1d_sum_reduce_with_program_cache(device, use_program_cache):
     dtype = ttl.tensor.DataType.BFLOAT16
 
     for _ in range(2):
+        H, W, latent = 32, 163840, 32
+        run_ssm_1d_sum_reduce(H, W, latent, dtype, mem_config, mem_config, device)
+        H, W, latent = 64, 163840, 32
         run_ssm_1d_sum_reduce(H, W, latent, dtype, mem_config, mem_config, device)
         dummy_shape = [1, 1, 32, 32]
         py_dummy_tensor = torch.randn(dummy_shape)
         tt_dummy_tensor = ttl.tensor.Tensor(py_dummy_tensor, dtype).to(ttl.tensor.Layout.TILE).to(device, mem_config)
 
-    assert device.num_program_cache_entries() == 1
+    assert device.num_program_cache_entries() == 2
```

Comment:
Why is there two?

Reply:
its generating two program cache for H=32 and H=64.